src/System.Runtime/tests/System/Reflection/AssemblyFlagsAttributeTests.cs

```diff
@@ -0,0 +1,61 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+#pragma warning disable 0618
+
+namespace System.Reflection.Tests
+{
+    public class AssemblyFlagsAttributeTests
+    {
+        [Theory]
+        [InlineData(AssemblyNameFlags.None, 0)]
+        [InlineData(AssemblyNameFlags.EnableJITcompileOptimizer, 16384)]
+        [InlineData(AssemblyNameFlags.EnableJITcompileTracking, 32768)]
+        [InlineData(AssemblyNameFlags.PublicKey, 1)]
+        [InlineData(AssemblyNameFlags.Retargetable, 256)]
+        [InlineData(int.MinValue, 2147483648)]
+        [InlineData((AssemblyNameFlags)(AssemblyNameFlags.None - 1), uint.MaxValue)]
+        [InlineData((AssemblyNameFlags)int.MaxValue, 2147483647)]
```

Comment:
Why pass in both assemblyFlags and expectedFlags?  Flags and AssemblyFlags both return the AssemblyNameFlags, just cast to uint and int, respectively.  We shouldn't need to hardcode the integral value into the test like this, nor duplicate it.

Reply:
I didn't want to simply copy the src implementation in the tests, but i fixed

Reply:
Thanks. Hardcoding the value was essentially copying the implementation, too... there's not much more you can do when the entire implementation is a simple cast.